lnrpc/routerrpc/router_server.go

```diff
@@ -627,12 +632,23 @@ func (s *Server) probePaymentRequest(ctx context.Context, paymentRequest string,
 // isLSP checks if the route hints indicate an LSP. An LSP is indicated with
 // true if the last node in each route hint has the same node id, false
```

Comment:
comment needs an update because we also check for public channels now.

Reply:
fixed